orbit-2-core/src/main/java/com/babylon/orbit2/RealContainer.kt

```diff
@@ -89,6 +91,7 @@ open class RealContainer<STATE : Any, SIDE_EFFECT : Any>(
     }
 
     companion object {
+        @ObsoleteCoroutinesApi
```

Comment:
alternative would be to mark `newSingleThreadContext(...)` with @Suppress("EXPERIMENTAL_API_USAGE") and not need the obsolete annotation here and at top of class